spec/controllers/graders_controller_spec.rb

```diff
@@ -1242,6 +1283,9 @@
                               criteria: [@criterion1.position, @criterion2.position, @criterion3.position],
                               graders: [@ta1.id, @ta2.id, @ta3.id], current_table: 'criteria_table' }
             expect(response.status).to eq(200)
+            @criterion1.reload
```

Comment:
I ended up adding `reload` to all of the `assign_ta` unit tests to avoid the data reload problem in the future.